packages/terra-pills/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "terra-pills",
```

Lets make this a scoped package
```suggestion
  "name": "@cerner/terra-pills",
```

---

changed here 5158250e39187534d43a615c7ee12bd447c7ab6b

---

changed here 5158250e39187534d43a615c7ee12bd447c7ab6b